enhancements/unmanaged-clair-database.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Unmanaged Clair Database
+authors:
+  - "@jonathankingfc"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2022-01-11
+last-updated: 2022-01-14
+status: provisional
+---
+
+# unmanaged-clair-database
+
+Currently, the operator allows only Quay's database to be unmanaged. There is business need to make Clair's database unmanaged as well, in the same manner as Quay's database now can be set to unmanaged, while still retaining Clair as a managed component. This also requires that a user has the ability to set a custom clair configuration (specifically, a custom database connection string)
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+> 1.  Should the clair config be fully customizable, or should it be limited to specific fields such as the DSN?
```

Comment:
I think we should allow any config here. We might choose the ones we want to validate (like the DSN for instance).

Reply:
Sounds good. The WIP implementation is currently doing this